src/ExpressionEvaluator/Core/Source/ResultProvider/Formatter.Values.cs

```diff
@@ -53,6 +53,12 @@ private string GetValueString(DkmClrValue value, DkmInspectionContext inspection
                 }
                 else if (lmrType.IsCharacter())
                 {
+                    // check if HostObjectValue is null, since any of these types might actually be a synthetic value as well.
+                    if (value.HostObjectValue == null)
```

Comment:
>value.HostObjectValue == null [](start = 24, length = 29)

Please consider writing tests for these cases (perhaps excluding `IntPtr` and `UIntPtr`). See `ValueFormattingTests` for examples that create and format `DkmClrValue` instances.

Reply:
Thank you, @cston ! Feedback addressed. Please review.